docs/application/dotnet/index.md

```diff
@@ -4,90 +4,24 @@ Tizen .NET is an exciting new way to develop applications for the Tizen operatin
 
 The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution. With Tizen .NET, you can use the C# programming language and the Common Language Infrastructure standards, and benefit from a managed runtime for faster application development and code execution that is efficient and secure.
 
-To start developing Tizen applications using .NET, see [Creating Your First Tizen .NET Application with Visual Studio](getting-started/first-app.md).
+The introduction to .NET applications documentation provides the overall information you need to become familiar with .NET-specific Tizen programming:
 
-<a id="runtime"></a>
-## Managed Runtime Advantages
+- [Creating Your First Tizen .NET Application with Visual Studio](getting-started/first-app.md)
 
-A managed runtime offers the following advantages to your application development:
+   Demonstrates how you can create an application using Visual Studio.
 
-- **Faster development**
+- [Managed runtime Advantages](getting-started/runtime.md)
 
-  Application development is accelerated since the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and Tizen Web API already provide this benefit.
+   Provides the benefits provided to your application development by the Managed runtime.
```

Comment:
Is it Managed runtime or managed runtime?
Please verify and update.

Reply:
'managed runtime' is correct.
I fix it.